qualtran/bloqs/block_encoding/lcu_block_encoding.py

```diff
@@ -127,9 +155,25 @@ def build_composite_bloq(self, bb: 'BloqBuilder', **soqs: SoquetT) -> Dict[str,
         soqs |= bb.add_d(self.select, **select_reg)
         return soqs
 
+    def wire_symbol(self, reg: Optional[Register], idx: Tuple[int, ...] = tuple()) -> 'WireSymbol':
+        if reg is None:
+            return Text('')
+        if reg.name == 'control':
+            return Circle(filled=bool(self.control_val))
+        else:
+            return TextBox('B[H]')
+
+    def get_single_qubit_controlled_bloq(self, control_val: int) -> 'LCUBlockEncoding':
```

Comment:
this should probably raise an error if `self.control_val` is not `None`.

Reply:
Done.